wiki/Community/Discord_servers/en.md

```diff
@@ -1,9 +1,89 @@
+
+### Project Loved
+
+<!-- This section as a whole needs to be discussed. Last time someone suggested listing the Loved suite of servers in the Project Loved wiki article, not everyone was on board. Moreover, some of its members and pepy themselves are not really happy on how everything is split apart. Question is whether we want to promote these servers while they exist, or ignore them in favour of the #osu-loved channel in the dev server -->
+
+[Project Loved](/wiki/Community/Project_Loved) organizes its workflows in different servers, divided by the ruleset which they specialize in. **The official development of the project is done in the osu! Discord server.** These hubs aim to facilitate the discussion of the mode's picks to be added. Non-game mode dependant discussions should always be carried out in `#osu-loved`, in the **osu!** Discord server.
+
+| Name | Owner |
+| :-- | :-- |
+| [osu! Project Loved](https://discord.gg/gn58Uk5sTE) | [Librarian](https://osu.ppy.sh/users/10083084) |
+| [osu!catch Project Loved](https://discord.gg/phgtyS4UCh) | [Wesley](https://osu.ppy.sh/users/2407265) |
+| [osu!mania Loved Community](https://discord.gg/Ededv7m) | [Paturages](https://osu.ppy.sh/users/1375479) |
+| [Project Loved: Taiko](https://discord.gg/GhfjtZ6) | [Hivie](https://osu.ppy.sh/users/14102976) |
+
+## Gameplay
+
+These servers are focused on giving users a place to discuss osu!'s main attraction — the gameplay! They serve as hubs for players to interact with their daily gaming ventures in osu!.
+
+| Name | Owner | Description |
+| :-- | :-- | :-- |
+| [osu! University](https://discord.gg/QubdHdnBVg ) | [DigitalHypno](https://osu.ppy.sh/users/4384207) | **osu! University** is a community Discord server with a focus on discussion about improvement, mainly in the [osu!](/wiki/Game_mode/osu!) game mode. It regularly hosts various events such as top player interviews, [tournaments](/wiki/Tournaments), and improvement-related studies, and is also well known for hosting paid coaching services. |
+| [osu! Medal Hunters](https://discord.gg/osumedalhunters) | [MegaMix](https://osu.ppy.sh/users/18152711) | **osu! Medal Hunters** is the main Discord server used by the medal hunting community, where people discuss [medals](/wiki/Medals) and [medal solutions](/wiki/Medals/Unlock_requirements). |
+
+## Development
+
+These servers focus on providing and developing tools and systems for the game. While the official development of osu! is done in the official community Discord server, certain projects require their own workspaces before proposing and merging their work in game.
+
+**The following servers are not officially affiliated with the game.** To follow up with official updates, join the [osu! Discord server](https://discord.gg/ppy) instead.
```

Comment:
this should already be obvious at this point in the article. if it's not, then I think you need to clarify earlier that there's only *one* official server

Reply:
Refer to the first comment. We could make it its own category at the forefront. Should avoid the repetition.